code/processes/idb.q

```diff
@@ -0,0 +1,45 @@
+/-default parameters
+\d .idb
+
+/- loads the idb and the sym file for given partition
+loaddb:{[pt]
+    ptdir: ` sv  .idb.savedir,(`$string[pt]),`;
```

Comment:
for better or worse, we generally don't use fully-qualified names in TorQ when within the same namespace

```suggestion
    ptdir: ` sv  savedir,(`$string[pt]),`;
```

(and similar elsewhere)